.travis.yml

```diff
@@ -12,7 +12,6 @@ branches:
 before_script:
 - npm install -g gulp
 - gulp
-script: npm test
```

Comment:
Why did you remove the testing part itself ?? 😕

Reply:
Travis defaults to this. No need to include. But if you want me to revert this I can